blog-homepage/src/missed-articles.js

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import missedArticlesData from './missed-articles.json';
+
+function LinkedImage(props) {
+  return(
+    <a className='missed-article link' href={props.source.link}>
+      <img className='missed-article image' src={props.source.image} alt={props.source.image}/>
+    </a>
+  )
+}
+
+function TitleDescriptionBlock(props){
+  return(
+    <div>
+     <h2 className='missed-article title'>{props.source.title}</h2>
+     <div className='missed-article description'>{props.source.description}</div>
+    </div>
+  )
+}
+
+function AuthorBio(props) {
+  const isMediumMember = props.source.isMediumMember;
+  return(
+  <div>
+   <h3 className='missed-article author-name'>{props.source.name}</h3>
+   <img
+    className='missed-article author-image'
+    style={ isMediumMember ? {border: '5px solid green'} : {}}
+    src={props.source.image}
+    alt={props.source.image}
+   />
+  </div>
+  )
+}
+
+function DateAndDuration(props) {
```

Comment:
I personally find these nested render functions within the same file somewhat hard to parse/understand. I guess it's personal preference, but I typically like to look in one place for the render function in a file to determine what the DOM output is going to look like.